code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -624,13 +624,18 @@
 	desc = "A special box for sensitive people."
 	icon_state = "hugbox"
 	foldable = null
+	var/cooldown = 0
 
 /obj/item/weapon/storage/box/hug/attack_self(mob/user)
-	..()
-	user.changeNext_move(CLICK_CD_MELEE)

```

Comment:
If I haven't made it clear, that proc is supposed to be a cooldown proc, to prevent spam/atatatata attacks.